docs/reference/eql/eql-search-api.asciidoc

```diff
@@ -89,6 +89,15 @@ request that targets only `bar*` still returns an error.
 +
 Defaults to `true`.
 
+`ccs_minimize_roundtrips`::
```

Comment:
We may want to link to https://www.elastic.co/guide/en/elasticsearch/reference/master/modules-cross-cluster-search.html#ccs-network-delays.

Reply:
👍 thanks a lot @jrodewig , I wasn't aware of this doc yet. That clarifies a lot!

Reply:
Right, the setting follows suite of the one for search and is similar to it, but I've hesitated to add the reference, since we don't actually link the two. We could add an explanation note to clarify the distinction, though.